test.sh

```diff
@@ -123,14 +123,9 @@ case $ENVIRONMENT_INDEX in
         run_test_docker Dockerfile_build_ubuntu64 test_64bit --test --junit-report
         rc=$?
         ;;
-    4)
-        # go1.12
-        run_test_docker Dockerfile_build_ubuntu64_go1.12 test_64bit --test --junit-report
-        rc=$?
-        ;;
-    "count")
-        echo $ENV_COUNT
-        ;;
+		"count")
```

Comment:
Looks like this is still set to tabs while the rest of this file uses spaces.

Reply:
actually fixed this time.